.semaphore/semaphore.yml

```diff
@@ -23,9 +23,13 @@ blocks:
           commands:
             - sem-version node 12
             - checkout
-            - cd ~/git/flowcrypt-ios/Core
-            - npm install
-            - npm test
+            - mkdir .custom-npm
+            - cd .custom-npm
+            - npm install npm@7.23.0
+            - cd ../Core
+            - ../.custom-npm/node_modules/.bin/npm --version
+            - ../.custom-npm/node_modules/.bin/npm install
+            - ../.custom-npm/node_modules/.bin/npm test
```

Comment:
I believe you can do sem-version npm 7.23.0 but I'm not 100% sure. Anyway if yes, that can be fixed later.